UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/SceneImporter/ImporterMeshes.cs

```diff
@@ -100,6 +181,136 @@ protected virtual async Task ConstructMesh(GLTFMesh mesh, int meshIndex, Cancell
 			await ConstructUnityMesh(unityData, meshIndex, mesh.Name);
 		}
 
+		/// <summary>
+		/// Populate a UnityEngine.Mesh from Draco generated SubMeshes
+		/// </summary>
+		/// <returns></returns>
+		protected async Task ConstructUnityMesh(GLTFMesh gltfMesh,DracoMeshLoader.DecodeResult[] decodeResults, Mesh.MeshDataArray meshes, int meshIndex, string meshName, bool requireTangents)
+		{
+			uint verticesLength = 0;
+			for (int i = 0; i < meshes.Length; i++)
+				verticesLength+= (uint)meshes[i].vertexCount;
+
+			Mesh mesh = new Mesh
+			{
+				name = meshName,
+#if UNITY_2017_3_OR_NEWER
+				indexFormat = verticesLength > 65535 ? IndexFormat.UInt32 : IndexFormat.UInt16,
+#endif
+			};
+
+			Mesh[] subMeshes = new Mesh[meshes.Length];
+			for (int i = 0; i < subMeshes.Length; i++)
+				subMeshes[i] = new Mesh();
+
+			Mesh.ApplyAndDisposeWritableMeshData(meshes, subMeshes);
+
+			CombineInstance[] combineInstances = new CombineInstance[subMeshes.Length];
+			for (int i = 0; i < combineInstances.Length; i++)
+			{
+				combineInstances[i] = new CombineInstance();
+				combineInstances[i].mesh = subMeshes[i];
+			}
+			mesh.CombineMeshes(combineInstances, false, false);
```

Comment:
Can you add a comment in the code that this is very slow and not ideal, and ideally a todo list / link to some issue to improve it in the future? Feels like a lot of the draco speedups are gone again if we're combining meshes again etc.